test/test_datatypes.py

```diff
@@ -1649,7 +1649,7 @@ def test31_anonymous_union(self):
         assert type(p.data_c[0]) == float
         assert p.intensity == 5.
 
-    @mark.xfail(condition=not IS_CLANG_REPL)
+    @mark.skipif(IS_CLANG_REPL==False, reason="Fails on Cling")
```

Comment:
```suggestion
    @mark.skipif(not IS_CLANG_REPL, reason="Fails on Cling")
```

Can we update all uses like that?

Reply:
Yep, it is done